File: Mobile AR/View Controllers/ViewController.m
//
//  ViewController.m
//  Mobile AR
//
//  Created by Justin Jean-Pierre on 2017-02-19.
//  Copyright © 2017 Jean-Pierre Digital. All rights reserved.
//

#import "ViewController.h"
#import "FileBrowser-swift.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];

    [self setTitle:@"Mobile AR"];

    if (self.navigationItem.rightBarButtonItem != nil) {
        [self.navigationItem.rightBarButtonItem setTarget:self];
        [self.navigationItem.rightBarButtonItem setAction:@selector(didPressShowFilesButton:)];
    }
}

-(IBAction)didPressShowFilesButton:(UIButton *) sender {
    [self presentViewController:[[FileBrowser alloc] init] animated:NO completion:^{}];
}

@end

File: Mobile AR/View Controllers/ViewController.swift
//
//  ViewController.swift
//  Mobile AR
//
//  Created by Justin Jean-Pierre on 2016-11-27.
//  Copyright © 2016 Jean-Pierre Digital. All rights reserved.
//

import UIKit
import FileBrowser

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()

        title = "Mobile AR"
   }

    @IBAction func didPressShowFilesButton(sender: UIButton) {
        presentViewController(FileBrowser(), animated: true, completion: nil)
    }
}
